lib/src/view/play/time_control_modal.dart

```diff
@@ -33,9 +35,48 @@ class TimeControlModal extends ConsumerWidget {
         child: ListView(
           shrinkWrap: true,
           children: [
-            Text(
-              context.l10n.timeControl,
-              style: Styles.title,
+            Row(
```

Comment:
This file should not be modified because it the one used by the create game screen. We do not want the clock to interfere with it.

You can reuse the code, simply copy/paste it to the `view/clock` folder. Code duplication is not an issue here.